GUI/src/app/pages/authorization-page/authorization-page.component.html

```diff
@@ -1,15 +1,12 @@
-  <div class="form-container">
-    <form class="authorization-form" [formGroup]="authorizationForm" (ngSubmit)="onSubmit()">
-      <h2 class="form-label">{{authorizationLabels.label}}</h2>
-      <br>
-      <input type="text" id="login-field" placeholder="{{authorizationLabels.userLogin}}" formControlName="login" />
-      <br>
-      <br>
-      <input type="password" id="password-field" placeholder="{{authorizationLabels.userPassword}}" formControlName="password" />
-      <br>
-      <button type="submit" [disabled]="!authorizationForm.valid" class="submit-button">{{authorizationLabels.enterButton}}</button>
-    </form>
-  </div>
-
-
-
+<div class="form-container">
+  <form class="authorization-form" [formGroup]="authorizationForm">
+    <h2 class="form-label">{{authorizationLabels.label}}</h2>
+    <br>
```

Comment:
please, remove br tags. Use styles to fix paddings

Reply:
Done.